### DISCUSSION TITLE
[RFC] playing is very slow in the mobile device. It's normal when playing in PC.

### DISCUSSION THREAD

**Discussion by trekGriffin** [stashapp/stash#4884] [Q&A] [answered]
I'm using the 0.25.1 version, the direct stream plays normally on the PC, but when using the android phone, it uses the webm standard format to play. Every 2 seconds, it needs to load, even when in the same WiFi environment as the PC. When I let it pause for buffering, it's normal. I don't know why this is happening.

**Comment by trekGriffin** [comment]
If someone is experiencing the same issue, it may be due to your hosted Stash server being a low-end server. Simply transfer it to a high-end server, and the problem should be solved.

**Comment by ghost** [comment]
I don't consider my setup a low-end server, and sometimes I have the same issue. It tends to be more common when I'm playing a video and, in another tab, I'm trying to view an image.
Just a note, I use it on iOS devices.

**Comment by trekGriffin** [comment, marked as answer]
I know the issue. It was just transcoding your video in the server side. That video is not compatible with your browser. If you switch to opera, the video is playing in direct stream mode. So it's video format issue. Maybe someone could develop a tool to convert these videos to a compatible format in the webpage.

**Reply by feederbox826** [reply]
In stash, under generate - there is an option to enable `Transcodes` - This will pre-convert the video into a more acceptable format at the cost of space
